### PR TITLE
[Antic] v0.1.0

### PR DESCRIPTION
**Pull request by thofma**

### CODE REVIEW

**Comment by thofma** [comment]
Hm, I don't understand the problem with the `LICENSE` file. He downloaded it correctly, but cannot find it later?

**Comment by giordano** [comment]
They are in `${WORKSPACE}/srcdir`.

**Comment by thofma** [comment]
I did it!

File: A/Antic/build_tarballs.jl [hunk 1]
```diff
@@ -38,7 +42,7 @@ products = [
 
 # Dependencies that must be installed before this package can be built
 dependencies = [
-    Dependency(PackageSpec(name="FLINT_jll", uuid="e134572f-a0d5-539d-bddf-3cad8db41a82"))
+    Dependency(PackageSpec(name="FLINT_jll", uuid="e134572f-a0d5-539d-bddf-3cad8db41a82", version=v"0.0.1+0"))
```
Comment (giordano) [inline A/Antic/build_tarballs.jl:45]: Don't use build numbers, `Pkg` doesn't like them:
```suggestion
    Dependency(PackageSpec(name="FLINT_jll", uuid="e134572f-a0d5-539d-bddf-3cad8db41a82", version=v"0.0.1"))
```

Comment (thofma) [reply A/Antic/build_tarballs.jl:45]: How does it distinguish between `+0` and `+1`?

Comment (giordano) [reply A/Antic/build_tarballs.jl]: Oops, didn't see this message before merging.  It doesn't, it'll grab the latest one :grimacing:

Comment (thofma) [reply A/Antic/build_tarballs.jl]: I won't matter here. It was just curiosity.

**Pull request merged**